edge/pkg/edged/edged.go

```diff
@@ -121,13 +121,18 @@ func (e *edged) Enable() bool {
 func (e *edged) Start() {
 	klog.Info("Starting edged...")
 
+	// FIXME: cleanup this code when the static pod mqtt broker no longer needs to be compatible
```

Comment:
Do we need to specify which version to cleanup here ? cc @fisherxu

Reply:
Write to changelog is also fine.